src/views/conference/2016/expect/expect.scss

```diff
@@ -172,7 +172,7 @@
             }
```

Comment:
Could you also have the line above:

`         @media only screen and (min-width: $mobile) and (max-width: $desktop - 1) {`

use the shorter query variables? Or maybe introduce a `$medium-and-intermediate` variable using this query?